SwiftUIDemoAppUITests/Tests/CallViewsTests.swift

```diff
@@ -35,8 +35,11 @@ final class CallViewsTests: StreamTestCase {
                 .setUserCount(participants)
                 .joinCall(callId, options: [.withCamera])
         }
-        AND("user enables grid view") {
-            userRobot.setView(mode: .grid)
+        AND("participant event label disappears & user enables grid view") {
+            userRobot
+                .waitForAppearanceOfParticipantEventLabel()
+                .waitForDisappearanceOfParticipantEventLabel()
+                .setView(mode: .grid)
```

Comment:
Why do we need to wait for the appearance and disappearance of the event label? 🤔

Reply:
The label isn't visible when we join the call so the disappearance exits immediately causing the test to fail